### DISCUSSION TITLE
Is this the right area to be asking general questions on git terminal commands

### DISCUSSION THREAD

**Discussion by dmastrop** [community/community#51028] [Programming Help] [answered]
### Select Topic Area

General

### Body

Is this the right area to ask general questions on git terminal commands
I have a GitHub remote repository, but I have very basic questions on committing, branching and merging.

warm regards

**Comment by airtower-luna** [comment]
This is *a* good place to ask, yes. The [Git mailing list or IRC](https://git-scm.com/community) are other options.

I also recommend looking at the [Git Book](https://git-scm.com/book/en/v2), chapters 2 and 3 cover the basics.

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for posting in the GitHub Community, @dmastrop and for asking such a relevant question! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
